kbase-extension/static/kbase/js/common/data.js

```diff
@@ -3,6 +3,7 @@ define(['kb_service/client/workspace', 'kb_service/utils', 'common/runtime'], (
     ServiceUtils,
     Runtime
 ) => {
+    'use strict';
```

Comment:
Some small eslint changes to this file, which came up in a search for jobs-related stuff